### ISSUE TITLE
A way to disable these logs to browser?

### ISSUE THREAD

**Issue by Maxiviper117** [mkkellogg/GaussianSplats3D#203]
![image](https://github.com/mkkellogg/GaussianSplats3D/assets/11472492/fa441444-6c19-4459-ac03-8a7b9d7353f6)

**Comment by mkkellogg** [comment]
Yeah I should probably make logging of this kind of information optional, and disable it by default.

**Comment by mkkellogg** [comment]
In the latest release [v0.3.9](https://github.com/mkkellogg/GaussianSplats3D/releases/tag/v0.3.9) I added a `logLevel` parameter to control the level of console logging. It now defaults to `None` :) Let me know if you see any issues with it.
